.github/workflows/grpc_source_sync.yml

```diff
@@ -0,0 +1,28 @@
+name: gRPC Native Source Shallow Sync Workflow
+on:
+  schedule:
+      - cron: '0 0 * * *' # Once per day at midnight
+  repository_dispatch:
+      types: [grpc-source-sync]
```

Comment:
just curious, what is `grpc-source-sync` and what does it do?

Reply:
this is the event type that we would use to trigger the source sync (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows). We can install the trigger from grpc/grpc as a way of detecting native source updates.

Reply:
Aha very cool! :)